pkgs/by-name/tt/ttaenc/package.nix

```
@@ -0,0 +1,44 @@
+{
+  stdenv,
+  lib,
+  fetchurl,
+}:
+stdenv.mkDerivation (finalAttrs: {
+  name = "ttaenc";
```

```suggestion
  pname = "ttaenc";
```
`name` is automatically generated based on pname and version.